.github/workflows/ci_linux_x64_clang_byollvm.yml

```diff
@@ -28,13 +28,14 @@ concurrency:
 jobs:
   linux_x64_clang_byollvm:
     runs-on: ubuntu-20.04
+    container: gcr.io/iree-oss/base@sha256:dc314b4fe30fc1315742512891357bffed4d1b62ffcb46258b1e0761c737b446
```

Comment:
I have similar changes planned for workflows like https://github.com/iree-org/iree/blob/main/.github/workflows/ci_linux_x64_clang_tsan.yml, but those aren't as simple... something about `docker_run.sh` seems to be load bearing there when it shouldn't be.